Documentation.md

```diff
@@ -0,0 +1,52 @@
+# Documentation
```

Comment:
Really good overall. I like the history in addition to the physics. Makes it feel more like reading a story.

Reply:
Thankyou so much.